### DISCUSSION TITLE
Hide first line added by LS to Emacs

### DISCUSSION THREAD

**Discussion by Zabrane** [erlang-ls/erlang_ls#1102] [Editors Customizations]
Hi @robertoaloi 

Would it be possible to hide the first line added by `erlang_ls` to Emacs? See this screenshot please.

![Screenshot_2021-10-02_at_09_19_41](https://user-images.githubusercontent.com/532544/135707576-a52f73ed-583c-4ad9-8822-1bb5be017b4a.png)

An option to hide it would be great, unless there's already a way to do it :-)
Thanks.

**Comment by Zabrane** [comment]
```lisp
(add-hook 'lsp-mode-hook
  (lambda()
    ;;; other stuff...
    (setq lsp-headerline-breadcrumb-enable nil)))
```

**Comment by robertoaloi** [comment]
Great that you found it. I just created a "discussion category" for these type of questions, so moving it there.

**Comment by robertoaloi** [comment]
You can either do that or:

```
(setq lsp-headerline-breadcrumb-mode nil)
```
I just documented this in the features part of the doc:

https://erlang-ls.github.io/features/
